### ISSUE TITLE
Fix aria-label for tooltips on org sign up flow's organization information page

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5129]
## Area or Component
Organization Sign up > Organization information

## Quick Summary
The `aria-label`s for the tooltips on that page are being rendered as "undefined tooltip" which is not a helpful description of the element's purpose.

### How to Replicate
Go to the organization sign up page (`/sign-up`) and then click "My organization is new to SimpleReport". On the "Organization information" page you will see that all the `aria-label`s for the tooltips on that page are "undefined tooltip".

## Accessibility Violation
[Name: The element's name is missing or incorrect
](https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect)

## Action Requested
Ensure the aria label makes sense. Looks like we are not passing a value for the `buttonLabel` attribute for the `TextWithTooltip` component on that page.

## Screenshots (optional)
![Screen Shot 2023-01-25 at 18.52.48.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/64459f24-ab42-4747-8ea7-15c351277200)

## Source Code (optional)

[which file or line is the issue attached to?]: #
